src/main/kotlin/com/github/kinquirer/core/AnsiOutput.kt

```diff
@@ -28,7 +29,14 @@ internal object AnsiOutput {
         }
 
         with(AnsiConsole.out()) {
-            write(rendered.toByteArray(UTF_8))
+            val consoleCP = Kernel32.INSTANCE.GetConsoleCP()
```

Comment:
In case you're wondering, getting the defaultCharset always returns UTF-8, which won't solve the problem, as the character encoding on a windows terminal is a specific codepage depending on the country. My computer uses Cp850 (Western locale, derived from Cp1252 but not the same character placement), and computers in other countries use other codepages.